iterm2_dwim/settings.py

```diff
@@ -0,0 +1,3 @@
+editor = <'emacs' OR 'sublime'>
```

Comment:
Even though the user is required to set this, how about we make it legal python code, e.g.

```python
editor = None  # Set this to 'emacs' or 'sublime'
```